src/js/components/tour/popover.tsx

```diff
@@ -1,62 +1,31 @@
-// import log from '@salesforce/design-system-react/utilities/log';
 import Button from '@salesforce/design-system-react/components/button';
-import IconSettings from '@salesforce/design-system-react/components/icon-settings';
 import Popover from '@salesforce/design-system-react/components/popover';
+import i18n from 'i18next';
 import React from 'react';
 
-import { User } from '~js/store/user/reducer';
-
 const TourPopover = ({
   align,
   heading,
   body,
-  isEnabled,
-  user,
 }: {
   align: string;
   heading: string;
   body: string;
-  isEnabled?: boolean;
-  user: User;
 }) => (
-  <IconSettings iconPath="static/images">
-    <Popover
-      align={align}
-      body={<p>{body}</p>}
-      footerWalkthroughActions={<Button variant="brand">Next</Button>}
-      heading={heading}
-      // isEnabled={user.self_guided_tour_enabled}
-      id="popover-walkthrough"
-      stepText="Step 2 of 4"
```

Comment:
@dvdherron I removed these footer/step items, because I don't think we want them for these individual popovers.